Packs/CommonTypes/ReleaseNotes/3_3_32.md

```diff
@@ -0,0 +1,6 @@
+
+#### Indicator Types
+- **emailRep**
+
+##### Email Type
+- Fixed an issue with the email better handle unicode points and other small issues.
```

Comment:
it's already known its a fix for the email indicators, :) please re-write the sentences.
also, specify what are the "small changes" includes

Reply:
How is it "already known"? `emailRep` is very unclear.

Reply:
not talking about the `emailrep` here but the Email Type